server/datastore/mysql/software_titles.go

```diff
@@ -337,7 +337,7 @@ GROUP BY st.id, package_self_service, package_name, package_version, package_url
 		vppAppsTeamsJoinCond = "FALSE"
 	}
 
-	additionalWhere := "TRUE"
+	additionalWhere := ""
```

Comment:
This default value may be replaced depending on one or more options, and it's a little cleaner to check for empty string than for "TRUE" when deciding what to append to the clause.